refaster-runner/src/main/java/tech/picnic/errorprone/refaster/runner/RefasterRuleSelector.java

```diff
@@ -320,12 +315,11 @@ static ImmutableList<UStatement> getTemplateStatements(BlockTemplate template) {
       return invokeMethod(METHOD_BLOCK_TEMPLATE_TEMPLATE_STATEMENTS, template);
     }
 
-    // Actually UClassIdent.
     static IdentifierTree getClassIdent(UStaticIdent tree) {
       return invokeMethod(METHOD_USTATIC_IDENT_CLASS_IDENT, tree);
     }
 
-    // XXX: Make nicer. Or rename the other params.
+    // Arguments to this method must actually be of the package-private type `UClassIdent`.
```

Comment:
One passes arguments _to_ a method. (Corresponding to the parameters _of_ said method. But here we are talking about the provided argument; the comment is here precisely because the parameter type isn't specific enough.)